tests/unit/test_service.py

```diff
@@ -1283,7 +1283,7 @@ def create_learning_path_for_tests(uow):
         student_id=1,
         course_id=1,
         topic_id=1,
-        algorithm="aco",
+        algorithm="ga",
```

Comment:
create a new test and dont delete the old test

Reply:
I agree and I added a new test

Reply:
I added it